src/mode.js

```diff
@@ -94,15 +97,17 @@ function getMode_(win) {
   return {
     localDev: isLocalDev,
     // Triggers validation
-    development: !!(developmentQuery['development'] == '1' ||
+    development: !!(hashQuery['development'] == '1' ||
         win.AMP_DEV_MODE),
     // Allows filtering validation errors by error category. For the
     // available categories, see ErrorCategory in validator/validator.proto.
-    filter: developmentQuery['filter'],
+    filter: hashQuery['filter'],
     /* global process: false */
     minified: !IS_DEV || process.env.NODE_ENV == 'production',
+    // Whether document is in an amp-lite viewer.
+    lite: searchQuery['amp_lite'] != undefined,
```

Comment:
This is currently what lite-viewer appends to Url, a value-less `amp_lite` param on the query string.